src/operator/ResourcesSearchbar.tsx

```diff
@@ -0,0 +1,26 @@
+// Libraries
+import React, {FC, useContext, ChangeEvent} from 'react'
+import {Input, IconFont} from '@influxdata/clockface'
+import {OperatorContext} from 'src/operator/context/operator'
+
+const ResourcesSearchbar: FC = () => {
+  const {activeTab, searchTerm, setSearchTerm} = useContext(OperatorContext)
+
+  const changeSearchTerm = (event: ChangeEvent<HTMLInputElement>) => {
+    setSearchTerm(event.target.value)
+  }
+
+  return (
+    <Input
+      icon={IconFont.Search}
+      placeholder={`Filter ${activeTab} by ${
+        activeTab === 'accounts' ? 'email' : 'id'
+      } ...`}
```

Comment:
This is going to have to be updated when we add a `Users` page later. The Quartz version of this injects a placeholder from the caller; can we keep that design here to avoid having this conditional grow in complexity over time?

I think that restoring the top-level routes will force separate top-level `Accounts` and `Organizations` pages which would then be a reasonable place for this prop to be set.